docs/configuration/options.md

```diff
@@ -140,9 +140,9 @@ jwt: {
   // Defaults to `session.maxAge`.
   maxAge: 60 * 60 * 24 * 30,
   // You can define your own encode/decode functions for signing and encryption
-  // if you want to override the default behaviour.
+  // if you want to override the default behavior.
   async encode({ secret, token, maxAge }) {},
-  async decode({ secret, token, maxAge }) {},
+  async decode({ secret, token }) {},
```

Comment:
From the [PR in core](https://github.com/nextauthjs/next-auth/pull/3039/files#diff-405c2a1f8acb4c30e45523ca6c4e62dae1f83da7e1028080448de989dbc5e73d) I can &nbsp;👀 that `encode` has kept `maxAge` as a param but `decode` not. I got initially a bit suspicious about this, so highlighting it here...

Reply:
Correct. There is no point in passing `maxAge` when decoding. You are just parsing (decoding) the token value.